includes/wc-conditional-functions.php

```diff
@@ -361,7 +361,7 @@ function wc_shipping_enabled() {
 	 * @return bool
 	 */
 	function wc_prices_include_tax() {
-		return wc_tax_enabled() && 'yes' === get_option( 'woocommerce_prices_include_tax' );
+		return wc_tax_enabled() && apply_filters( 'wc_prices_include_tax', get_option( 'woocommerce_prices_include_tax' ) === 'yes' );
```

Comment:
Need to fix the filter name, all filters and actions should start with `woocommerce_` and not `wc_`.

Reply:
Also missing changelog in this PR.

Reply:
done